README.md

```diff
@@ -16,17 +16,37 @@ To guide development we [have documented the user needs](docs/NEEDS.md) and [ass
 
 ## Usage
 
+To run a GOV.UK application, `make` it first (you only need to do this once per application):
+
 ```sh
-# Run a Rake task on Whitehall
-whitehall$ govuk-docker run bundle exec rake -T
+# This assumes you're in the ~/govuk/govuk-docker directory.
+$ make content-tagger
+```
+
+Then run it with `startup`:
 
+```sh
 # Start content-tagger including dependencies. Visit it at content-tagger.dev.gov.uk
-content-tagger$ govuk-docker startup
+$ govuk-docker startup --service=collections-publisher
+```
 
-# Start content-publisher plus an "end-to-end" stack
-content-publisher$ govuk-docker startup e2e
+Alternatively, you can run `startup` directly from the application directory. If no `service` parameter is provided, govuk-docker matches the name of the current directory with the [corresponding docker-compose.yml in govuk-docker/services](https://github.com/alphagov/govuk-docker/blob/master/services/content-tagger/docker-compose.yml):
+
+```sh
+$ cd ~/govuk/content-tagger && govuk-docker startup
 ```
 
+`govuk-docker startup` runs the application on the `app` stack by default, but you can [override the stack](#stacks)) if needed.
+
+Another useful govuk-docker command is `run`:
+
+```sh
+# Run a Rake task on Whitehall
+$ cd ~/govuk/whitehall && govuk-docker run bundle exec rake -T
+```
+
+For a full list of govuk-docker commands, run `govuk-docker help`.
+
 ## Installation
 
 ### Prerequisites
```

Comment:
I think we can drop the prerequisites, as they're installed by `govuk-docker setup`?

Reply:
I've given it an update 👍